core/src/main/java/org/apache/accumulo/core/file/blockfile/impl/ScanCacheProvider.java

```diff
@@ -30,42 +32,45 @@ public class ScanCacheProvider implements CacheProvider {
 
   public ScanCacheProvider(AccumuloConfiguration tableConfig, ScanDispatch dispatch,
       BlockCache indexCache, BlockCache dataCache) {
+
+    var loggingIndexCache = LoggingBlockCache.wrap(CacheType.INDEX, indexCache);
+    var loggingDataCache = LoggingBlockCache.wrap(CacheType.DATA, dataCache);
```

Comment:
I'm curious if we should only do this if trace is enabled for the LoggingBlockCache logger.

Reply:
Inside the `LoggingBlockCache.wrap` method it checks if trace is enabled and if its not it just returns the cache w/o wrapping it.

Reply:
Ok, thanks, I missed that.